doc/userDocs/usersGuide.md

```diff
@@ -141,29 +141,21 @@ The image below shows the lines that need to be edited in the configuration file
 3. Update *kubeconfigContext* parameters with the name of the context for your cluster from the kubeconfig file, such as `kubernetes-admin@kubernetes`.
     - Set an empty string (`""`) to use the current-context defined in the kubeconfigFile.
 4. Choose a value for *appIngressMethod* that reflects the method you want to use
-  for communication between the workload drivers and the applications.
-    - If the workload drivers and applications are running on different Kubernetes 
-      clusters, you can choose either:
-        - `loadbalancer`:  Traffic from the workload drivers to the applications will
-          use the external IP address of a Kubernetes LoadBalancer service. Your cluster must 
-          support provisioning external IP addresses for LoadBalancer services. 
-        - `nodeport`:  Traffic from the workload drivers to the applications will
-          use the external IP addresses of the Kubernetes Nodes via a NodePort service. Your cluster must 
-          support exposing the node IP addresses externally using NodePort services.
-    - If the workload drivers and the applications are running on the same Kubernetes 
-      cluster, you can choose among:
-        - `none`: Traffic from the workload drivers to the applications will use a 
-          cluster internal address provisioned by a Kubernetes ClusterIP service.  
-          This method will work on all clusters.
-        - `loadbalancer`: Traffic from the workload drivers to the applications will
+  for communication between the workload drivers and the applications. You can choose 
+  from among:
+    - `clusterip`: Traffic from the workload drivers to the applications will use a 
+          cluster internal address provisioned by a Kubernetes ClusterIP service.  This 
+          choice will only work if the workload drivers and applications are running 
+          on the same cluster.
```

Comment:
I forgot to ask, what happens if you specify clusterip with multiple clusters?
Is there a check, or does it just run into errors, or?

Reply:
Good question.  There should be a check but there is not.  Currently it will just not work, and I don't know what errors it will give (as I only have a single cluster).  I will add a check.